site/content/docs/main/aggregator-permissions.md

```diff
@@ -0,0 +1,27 @@
+# Aggregator Permissions
+
+Through `sonobuoy gen` command you can customize multiple aspects of the run in your cluster. In these multiple aspects you can also customize aggregator permissions of a user in your cluster. These permissions act as limitations, it determines that in what tasks a user can take part in the run. We can customize this aspect of the run through `aggregator-permissions` flag.
+
+- Run `sonobuoy gen` without changing aggregator permissions, value of parameter `AggregatorPermissions` is `clusterAdmin`in sonobuoy config.
+
+![aggregator-permissions1](img/aggregator-permissions1.png)
+
+- After running `sonobuoy gen --aggregator-permissions clusterRead`, value of parameter `AggregatorPermissions` gets changed to `clusterRead`.
+
+![aggregator-permissions1](img/aggregator-permissions2.png)
+
+## Type of Aggregator Permissions
+
+Allowable values are `[namespaced, clusterAdmin, clusterRead]`, `clusterAdmin` is default value.
+
+### clusterAdmin
+
+- `clusterAdmin` is the default value. With this value a user can do pretty much everything in the run, it does not implement any restrictions. Most of these are required for the e2e conformance tests to work since they create/destroy namespaces, pods etc.
+
+### namespaceAdmin
+
+- `namespaceAdmin` is just ability to do CRUD operations but in a single namespace. Sonobuoy does not create the namespace so it needs to already exist and you have to run `--skip-preflight` to avoid Sonobuoy complaining about that. `e2e tests` not work in this mode, they won't even start because it has startup logic that requires looking in kube-system and checking nodes. Daemonsets will also not work in this mode because we monitor them in a way that we require GET on /nodes so that we know which nodes were responding with results. Since we have NO cluster permissions, we can't get nodes, so we can't know which results to get or wait for, so we just ignore daemonsets.
```

Comment:
I think maybe this will flow better if we start with the motivation of what this is for and then list the limitations.

```
namespaceAdmin is the most restrictive preset permissions Sonobuoy provides and ensures that Sonobuoy and its plugins do not impact other namespaces at all.

Due to these limitations there are a number of things to note:
 - Sonobuoy does not create the namespace so it needs to already exist
 - You must provide `--skip-preflight` to avoid Sonobuoy from complaining about the preexisting namespace
 - The `e2e` plugin (conformance tests) will not work in this mode and won't even start up due to severely limited permissions
 - Daemonset plugins will not work in this mode because Sonobuoy monitors them on a per-node basis. Since Sonobuoy can't query the list of nodes in the cluster, it can't properly monitor or gather results from them. At this time, Daemonset plugins will simply be ignored.
```

Reply:
Notice that instead of writing `we` I use `Sonobuoy` whenever possible.